zebedee-cms/src/main/java/com/github/onsdigital/zebedee/keyring/KeyringMigrationImpl.java

```diff
@@ -1,10 +1,15 @@
 package com.github.onsdigital.zebedee.keyring;
 
 import com.github.onsdigital.zebedee.model.Collection;
+import com.github.onsdigital.zebedee.model.KeyringCache;
+import com.github.onsdigital.zebedee.model.encryption.ApplicationKeys;
+import com.github.onsdigital.zebedee.session.model.Session;
+import com.github.onsdigital.zebedee.session.service.Sessions;
 import com.github.onsdigital.zebedee.user.model.User;
 import org.apache.commons.lang3.StringUtils;
 
 import javax.crypto.SecretKey;
+import java.io.IOException;
 
 /**
  * KeyringMigrationImpl is an abstraction in front of the existing keyring funtionality. This abstraction enables us to
```

Comment:
Just wondering if some of the tenses in this JavaDoc need updating depending on progress?
Eg. "When the new central keyring has been built". Not sure if that's now or not.

Reply:
Yeah that is a fair comment - the new central keyring hasn't been fully implemented yet. This class isn't the new central keyring, its a mechanism that allows us to decouple the existing keyring implementation from the classes depending on it. Putting this layer in will allow us to  switch to the new implementation when it's ready without breaking anything -Confusing I know.

I added this javadoc to try and explain what it is we're doing to aid anyone trying to make sense of it if they haven't been involved in the planning/design. But I take you point and I will make sure this is updated to reflect the current state of things when it changes 👍